docs/modules/ROOT/pages/how-tos/enable-multi-forwarder.adoc

```diff
@@ -0,0 +1,70 @@
+= Enable Multi LogForwarder
+
+Red Hat OpenShift Logging Operator only watches the `openshift-logging` namespace.
+If you want the Red Hat OpenShift Logging Operator to watch all namespaces on your cluster, you must redeploy the Operator.
+You can complete the following procedure to redeploy the Operator without deleting your logging components.
```

Comment:
I assume you've tested that this procedure actually doesn't delete the logging stack?

Reply:
Not yet, but it is what the upstream documentation claims :D The plan was to test it with the master branch on the Lab.

Reply:
Confirmed to work 👍